MAUI/ProgressBar/Appearance.md

```diff
@@ -0,0 +1,390 @@
+
+<progressBar:SfLinearProgressBar Progress="100" >
+    <progressBar:SfLinearProgressBar.GradientStops>
+        <progressBar:ProgressGradientStop Color="#88A0D9EF" Value="0"/>
+        <progressBar:ProgressGradientStop Color="#AA62C1E5" Value="25"/>
+        <progressBar:ProgressGradientStop Color="#DD20A7DB" Value="50"/>
+        <progressBar:ProgressGradientStop Color="#FF1C96C5" Value="75"/>
+    </progressBar:SfLinearProgressBar.GradientStops>
+</progressBar:SfLinearProgressBar>
+
+<!--Using the circular progress bar-->
+
+<progressBar:SfCircularProgressBar Progress="100">
+    <progressBar:SfCircularProgressBar.GradientStops>
+        <progressBar:ProgressGradientStop Color="#88A0D9EF" Value="0"/>
+        <progressBar:ProgressGradientStop Color="#AA62C1E5" Value="25"/>
+        <progressBar:ProgressGradientStop Color="#DD20A7DB" Value="50"/>
+        <progressBar:ProgressGradientStop Color="#FF1C96C5" Value="75"/>
+    </progressBar:SfCircularProgressBar.GradientStops>
+</progressBar:SfCircularProgressBar>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+// Using the linear progress bar
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.Progress = 100;
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("88A0D9EF"), Value = 0 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("AA62C1E5"), Value = 25 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("DD20A7DB"), Value = 50 });
+linearProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("FF1C96C5"), Value = 75 });
+this.Content = linearProgressBar;
+
+// Using the circular progress bar
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.Progress = 100;
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("88A0D9EF"), Value = 0 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("AA62C1E5"), Value = 25 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("DD20A7DB"), Value = 50 });
+circularProgressBar.GradientStops.Add(new ProgressGradientStop { Color = Color.FromArgb("FF1C96C5"), Value = 75 });
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![.NET MAUI ProgressBar with gradient range](images/appearance/gradient.png)
+
+## Thickness
+
+### Linear progress bar
+
+In the linear progress bar, the height of the track, progress, and secondary progress can be customized using the `TrackHeight`, `ProgressHeight`, and `SecondaryProgressHeight` properties, respectively.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<progressBar:SfLinearProgressBar Progress="100" 
+                                 TrackHeight="10" 
+                                 ProgressHeight="10"
+                                 SecondaryProgressHeight="10" />
```

Comment:
Apply Progress as 30 and secondary progress as 70, then only we can see all element's height in the output.

Reply:
done